txwrapplog.py

```diff
@@ -65,6 +72,10 @@ def wrapped(event):
     return wrapped
 
 
+def _timestamp():
```

Comment:
I needed to mock the default datetime.utcnow(), but in python you can't mock built-in classes.

So i needed to use an outside function for it to work.